esp_schedule/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+set(component_srcs "src/esp_schedule.c"
+                   "src/esp_schedule_nvs.c")
+
+idf_component_register(SRCS "${component_srcs}"
+                       INCLUDE_DIRS "include"
+                       PRIV_INCLUDE_DIRS "src"
+                       PRIV_REQUIRES nvs_flash esp_netif)
+
+target_compile_options(${COMPONENT_LIB} PRIVATE -Wno-unused-function)
```

Comment:
why not fix the "unused-function" warning since the source code are all in our control?